docs/guides/primitive-input-output.ipynb

```diff
@@ -0,0 +1,284 @@
+    "# using the set of parameters and observables.\n",
+    "estimator = Estimator(mode=backend)\n",
+    "job = estimator.run([estimator_pub])\n",
+    "result = job.result()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dfbaa52f-be7a-406d-b1d7-950daf90fd7f",
+   "metadata": {},
+   "source": [
+    "## Overview of primitive results\n",
+    "\n",
+    "Once one or more PUBs are sent to a QPU for execution and a job successfully completes, the data is returned as a `PrimitiveResult` container object accessed by calling the `RuntimeJobV2.result()` method. This data structure contains an iterable list of `PubResult` objects - one for every PUB - and each of these `PubResult` objects then contains the execution results of the PUB that was submitted. The data for each `PubResult` is contained within a `DataBin` container object, the attributes of which are dependent on the structure of the associated PUB, as well as the error mitigation options specified by the primitive that the job was submitted with (for example, [ZNE](./error-mitigation-and-suppression-techniques#zero-noise-extrapolation-zne) or [PEC](./error-mitigation-and-suppression-techniques#probabilistic-error-cancellation-pec)).\n",
+    "\n",
+    "However, no matter what runtime options are specified, each `PubResult` for the Estimator primitive contains a list of expectation values (`PubResult.data.evs`) and associated standard deviations (either `PubResult.data.stds` or `PubResult.data.ensemble_standard_error` depending on the `resilience_level` used)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 23,
+   "id": "8a4a1929-6191-4061-9fb2-916d2580d1ae",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The result of the submitted job had 1 PUB and has a value:\n",
+      " PrimitiveResult([PubResult(data=DataBin(evs=np.ndarray(<shape=(3, 100), dtype=float64>), stds=np.ndarray(<shape=(3, 100), dtype=float64>), ensemble_standard_error=np.ndarray(<shape=(3, 100), dtype=float64>), shape=(3, 100)), metadata={'shots': 8192, 'target_precision': 0.015625, 'circuit_metadata': {}, 'resilience': {}, 'num_randomizations': 64})], metadata={'dynamical_decoupling': {'enable': False, 'sequence_type': 'XX', 'extra_slack_distribution': 'middle', 'scheduling_method': 'alap'}, 'twirling': {'enable_gates': False, 'enable_measure': True, 'num_randomizations': 'auto', 'shots_per_randomization': 'auto', 'interleave_randomizations': True, 'strategy': 'active-accum'}, 'resilience': {'measure_mitigation': True, 'zne_mitigation': False, 'pec_mitigation': False}, 'version': 2})\n",
+      "\n",
+      "The associated PubResult of this job has the following data:\n",
+      " DataBin(evs=np.ndarray(<shape=(3, 100), dtype=float64>), stds=np.ndarray(<shape=(3, 100), dtype=float64>), ensemble_standard_error=np.ndarray(<shape=(3, 100), dtype=float64>), shape=(3, 100))\n",
```

Comment:
could we show users how to drill down into this data a bit more, rather than just printing out `np.array(....`

Reply:
Hopefully I addressed this! Feel free to give it another read.